### PR TITLE
fix(log_fmt_console): double option in config

### PR DESCRIPTION
**Pull request by ze42**
When option was specified, it appeared twice in config, making configuration
file buggy, and not loaded.

### PR progress checklist (to be filled in by reviewers)

- [ ] Changes to documentation are appropriate (or tick if not required)
- [ ] Changes to tests are appropriate (or tick if not required)
- [ ] Reviews completed

---

#### Primary type

- [ ] `[build]`    Changes related to the build system
- [ ] `[chore]`    Changes to the build process or auxiliary tools and libraries such as documentation generation
- [ ] `[ci]`       Changes to the continuous integration configuration
- [ ] `[feat]`     A new feature
- [x] `[fix]`      A bug fix
- [ ] `[perf]`     A code change that improves performance
- [ ] `[refactor]` A code change that neither fixes a bug nor adds a feature
- [ ] `[revert]`   A change used to revert a previous commit
- [ ] `[style]`    Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc.)

#### Secondary type

- [ ] `[docs]`     Documentation changes
- [ ] `[test]`     Adding missing or correcting existing tests

### Does this PR introduce a `BREAKING CHANGE`?

No.

### Describe the changes you're proposing

log_fmt_console can now be specified without breaking the generated config files.

### Pillar / config required to test the proposed changes

```
salt:
  master:
    log_fmt_console: '%(colorlevel)s %(colormsg)s'
  minion:
    log_fmt_console: '%(colorlevel)s %(colormsg)s'
```

### Documentation checklist

- [ ] Updated the `README` (e.g. `Available states`).
- [ ] Updated `pillar.example`.

### Testing checklist

- [ ] Included in Kitchen (i.e. under `state_top`).
- [ ] Covered by new/existing tests (e.g. InSpec, Serverspec, etc.).
- [ ] Updated the relevant test pillar.

### CODE REVIEW

File: salt/files/master.d/f_defaults.conf [hunk 1]
```diff
@@ -1590,7 +1590,7 @@ mine_get:
 # the coloring of the messages, these color formatters also include padding as
 # well.  Color LogRecord attributes are only available for console logging.
 #
-{{ get_config('log_fmt_console', "'%(colorlevel)s %(colormsg)s'") }}
+#log_fmt_console "%(colorlevel)s %(colormsg)s"
 {{ get_config('log_fmt_console', "'[%(levelname)-8s] %(message)s'") }}
```
Comment (noelmcloughlin) [inline salt/files/master.d/f_defaults.conf:1594]: Line 1594 looks wrong too. I think 'messages' should be 'colormsg' ?

File: salt/files/minion.d/f_defaults.conf [hunk 2]
```diff
@@ -1080,7 +1080,7 @@ ext_pillar:
 # the coloring of the messages, these color formatters also include padding as
 # well.  Color LogRecord attributes are only available for console logging.
 #
-{{ get_config('log_fmt_console', "'%(colorlevel)s %(colormsg)s'") }}
+#log_fmt_console "%(colorlevel)s %(colormsg)s"
```
Comment (noelmcloughlin) [inline salt/files/minion.d/f_defaults.conf:1083]: Line looks wrong too. I think 'message' should be 'colormsg' ?

File: salt/files/master.d/f_defaults.conf (hunk 1 above)
Comment (ze42) [reply salt/files/master.d/f_defaults.conf:1594]: No, default is the log level and message, without colors.

Previously:
* not specifying a config gives 2 comment exemples, one with color format, one without color
* specifying a config would give 2 lines with the specified value, and fail to load config

Now, it always get the color message as comment, and could have along the non-color commented, or whatever is specified as format message.

File: salt/files/minion.d/f_defaults.conf (hunk 2 above)
Comment (ze42) [reply salt/files/minion.d/f_defaults.conf:1083]: see other comment.

**Review by aboe76 — APPROVED** [review]

**Comment by noelmcloughlin** [comment]
Thanks @ze42  and @aboe76  for review

**Pull request merged**